x/leverage/keeper/oracle.go

```diff
@@ -12,6 +12,9 @@ import (
 	oracletypes "github.com/umee-network/umee/v6/x/oracle/types"
 )
 
+// TODO: parameterize this
+const MaxSpotPriceAge = 180 // 180 seconds = 3 minutes
```

Comment:
@robert-zaremba This is the time limit on prices used for transactions.

Prices older than this are considered "Expired", and can only be used for queries.